config/settings.yml

```
@@ -854,6 +854,7 @@
       - VERSION
     :ping_depot: true
     :ping_depot_timeout: 20
+  :filter: []
```

`filter` seems a bit generic here, so maybe `password_filter`?  Not sure on the best name.

```suggestion
  :password_filter: []
```

---

@Fryguy so I didn't go with `password_filter` because other things might also make sense that are "password adjacent", but not technically passwords:

- keys
- tokens
- secrets

So maybe "secret_filter" is a better name?  Thoughts?

---

Oh yeah good call - `secret_filter` sounds nice